action/steps/activate.sh

```diff
@@ -24,7 +26,7 @@ if [[ -n "$UNITY_LICENSE" ]] || [[ -n "$UNITY_LICENSE_FILE"  ]]; then
     echo "$UNITY_LICENSE" | tr -d '\r' > $FILE_PATH
   elif [[ -n "$UNITY_LICENSE_FILE" ]]; then
     # Copy license file from file system
-    cat "$UNITY_LICENSE_FILE" | tr -d '\r' > $FILE_PATH
+    cat "$UNITY_PROJECT_PATH/$UNITY_LICENSE_FILE" | tr -d '\r' > $FILE_PATH
```

Comment:
This effectively locks files our of the home directory, which would also be a better place to put this file, than in the project directory itself.

I believe this broke due to https://github.com/game-ci/unity-test-runner/pull/109, although I do not see you referencing unity-test-runner@main, could you please confirm?

If indeed it's caused by that, a better solution might be simply move line 5~7 to below this block and prefix FILE_PATH with a slash to put it in the root.

Reply:
Yes. This broke due to #109 
My license file is placed in the project directory.

Reply:
In that case I would suggest the solution in the last paragraph, so that the action also keeps working for people who use the profile folder for storing the license file.

Reply:
Yea. But this has two condition:

Active by license file.
Active by serial number.
So. If move line 5~7 to condition block, then will be to copy them two condition blocks.

Reply:
This is not what was meant by that suggestion.

You can move line 5~7 _below_ the condition blocks instead of changing the path where it looks for the license.

We could use an absolute path for the license file (stored in `$FILE_PATH` variable).